components/brave_extension/extension/brave_extension/braveShieldsPanel.tsx

```diff
@@ -18,13 +20,14 @@ chrome.storage.local.get('state', (obj) => {
   const store: any = new Store({
     portName: 'BRAVE'
   })
-
   store.ready()
     .then(() => {
       const mountNode: HTMLElement | null = document.querySelector('#root')
+      const theme: ThemeType = store.state.shieldsPanel.theme
```

Comment:
Correct me if I'm wrong here, but reading from state here will never result in an update when state changes, so it's not serving a useful data-flow purpose?